### DISCUSSION TITLE
IAC DO(NT) ECHO

### DISCUSSION THREAD

**Discussion by nyankers** [scandum/tintin#90] [Bugs]
Hey! While working on something with a friend, we noticed some peculiar behavior from tintin++. Namely, it sends IAC DO ECHO but does not send IAC DONT ECHO. Looking at the code, I see the latter is just straight up commented out. The debug message for both says (SKIPPED) but the former is not in fact skipped.

Something about this seems a bit funny, so I thought I'd ask in case there's a bug here.

**Comment by scandum** [comment]
There was a mud creating infinite echo loops, so I patched the echo code. I'll fix the (SKIPPED) message for IAC DO ECHO though.
